### PR TITLE
asciidoc: Fix checksums error due to stealth upgrade

### PR DESCRIPTION
**Pull request by jhoyt4**
###### Type(s)

- [X] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.1 20D74
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

Reference trac ticket 62273 (https://trac.macports.org/ticket/62273)

### CODE REVIEW

**Comment by ctreleaven** [comment]
Hi John:

One more detail to take care of...see https://trac.macports.org/wiki/PortfileRecipes#stealth-updates

**Review by neverpanic — CHANGES_REQUESTED** [review]
Please add dist_subdir, otherwise users may or may not end up with our mirrored copy and experience checksum mismatches.

**Comment by jhoyt4** [comment]
I've just pushed an update that includes rev'ing to 9.1.0 and adds both a version and dist_subdir to the portfile.

**Comment by jhoyt4** [comment]
That last force push was to remove a redundant version number in the portfile

File: textproc/asciidoc/Portfile [hunk 1]
```diff
@@ -4,13 +4,16 @@ PortSystem          1.0
...
 revision            0
-checksums           rmd160  8940c0c8b830f43c97cf163a28bf1bcef31dd173 \
-                    sha256  71acadb505bc34971cdc78a1555e342dfe6da0f0f78b173d96d0dd8519083487 \
-                    size    1119595
+checksums           rmd160  4fef2148ed93859afe13031e0c953bbe0dabba29 \
+                    sha256  98941741bcef1b172bce735c9393255c095fe88642ac725a43a32afda0d73a4a \
+                    size    1131368
 name                asciidoc
 
+dist_subdir         ${name}/${version}
```
Comment (neverpanic) [inline textproc/asciidoc/Portfile:15]: Right, but when updating the version, the stealth upgrade procedures do no longer apply, so the `dist_subdir` is unnecessary and should not be set.

Sorry for the confusion.

**Review by neverpanic — CHANGES_REQUESTED** [review]

File: textproc/asciidoc/Portfile (hunk 1 above)
Comment (jhoyt4) [reply textproc/asciidoc/Portfile]: All good - learning as I go, so I appreciate the patience.  It should be fixed in the Portfile now

**Comment by jhoyt4** [comment]
...well, it looks like someone else just took care of this in the repo https://github.com/macports/macports-ports/commit/8d6bc182845d20e7dddeb5a9b28b0073673a2027

**Comment by jhoyt4** [comment]
Close since @larryv committed their own version...

Someone should close the trac ticket to reflect his as resolved...

**Comment by neverpanic** [comment]
Done, thanks for your help anyway!

**Pull request closed without merging**